### PR TITLE
WIP Initial Version of Frontend

### PR DESCRIPTION
**Pull request by ghost**
## Motivation
This pull request is an implementation for the AskInternally design on Figma. 

## Left to be Done
1. Add background ellipses **(done)**
2. Update Footer with appropriate information **(removed footer for now)**
3. Update FAQ questions with proper answers **(done)**

## Images
image

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -183,144 +168,36 @@ const metadata = {
...
-      },
-      {
-        title: 'Community support',
-        description: `Join the growing AstroWind community for insights, resources, and assistance, ensuring you're never alone on your web development journey.`,
+          "Confident in our platform's potential, we aim to achieve a similar or higher valuation within the next 12-18 months.",
       },
     ]}
     image={{
-      src: 'https://images.unsplash.com/photo-1611462985358-60d3498e0364?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2070&q=80',
+      src: 'https://s3-alpha-sig.figma.com/img/075b/3461/10f664a176f07d4a2601d540781321ba?Expires=1716163200&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=S-2IlVoUICKnHQdO~1w2nMUMGvvVSzE4wR7OeJp1StKSCyaaZ8iT9gRMYXxLLlX6gfrRnCEqL7jpRyJnXf4gmQ4Q2dXq9oyCyouxEDwvKdh4Xz4vT6nwUANkrztH-~YAbRSz~A5D9AbXUgZ1D8QNVw9AZzwB5qPEvaH-sMawgfy6xFb-IxaI0z3efYVSF4lC5SK1jFV05nJ2qDx41JHmJFvbjLLxUd7FNwjldgQ-XiDvYOZ5Kq6o131drNDAu29Clog2Ghsb5rzOgbNov~8g1BmwyjuBLYILmokW0sxon1bPcX80CFz5-fQxNJUXVyYyzYzceS1XokY88nyLjyEFjg__',
```
Comment (sunapi386) [inline src/pages/index.astro]: Hm, this seems to expire. Notice the Expires=1716163200, it will break in prod.

File: src/pages/index.astro [hunk 2]
```diff
@@ -341,59 +218,26 @@ const metadata = {
...
+        title: "What is something that you would like to try again?",
         description:
-          "A business big enough that it could be listed on the NASDAQ goes belly up. Disappears! It ceases to exist without me. No, you clearly don't know who you're talking to, so let me clue you in.",
+          "A flower in my garden, a mystery in my panties. Heart attack never stopped old Big Bear. I didn't even know we were calling him Big Bear.",
       },
       {
         title: 'If you could only ask one question to each person you meet, what would that question be?',
         description:
-          "This is not about revenge. This is about justice. A lot of things can change in twelve years, Admiral. Well, that's certainly good to know. About four years. I got tired of hearing how young I looked.",
+          "A business big enough that it could be listed on the NASDAQ goes belly up. Disappears! It ceases to exist without me. No, you clearly don't know who you're talking to, so let me clue you in.",
```
Comment (sunapi386) [inline src/pages/index.astro]: These text still needs to be changed

**Review by sunapi386 — CHANGES_REQUESTED** [review]

File: src/pages/index.astro [hunk 3]
```diff
@@ -183,144 +173,36 @@ const metadata = {
...
-      },
-      {
-        title: 'Community support',
-        description: `Join the growing AstroWind community for insights, resources, and assistance, ensuring you're never alone on your web development journey.`,
+          "Confident in our platform's potential, we aim to achieve a similar or higher valuation within the next 12-18 months.",
       },
     ]}
     image={{
-      src: 'https://images.unsplash.com/photo-1611462985358-60d3498e0364?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2070&q=80',
+      src: 'https://i.ibb.co/k89md15/10f664a176f07d4a2601d540781321ba.png',
```
Comment (sunapi386) [inline src/pages/index.astro:192]: Raw linking to other's resources such as the png here is a no-no and frowned on. Essentially they pay for hosting and it's prone to breakage. Download all content into our assets folder and load all content from our own servers.

Future todo: Replace this with appropriate image (check with designer) and add to the asset folder.

File: src/pages/index.astro [hunk 4]
```diff
@@ -331,69 +213,26 @@ const metadata = {
...
         description:
-          "Well, the way they make shows is, they make one show. That show's called a pilot. Then they show that show to the people who make shows, and on the strength of that one show they decide if they're going to make more shows.",
+          'With the rise of AI and LLMs, there exists a growing cocnern about using LLMs with private and confidential data. This gives rise to the need for offline AI solutions.',
       },
       {
-        title: "What's something that you don't understand?",
+        title: 'Why AskInternally?',
         description:
-          "A flower in my garden, a mystery in my panties. Heart attack never stopped old Big Bear. I didn't even know we were calling him Big Bear.",
+          "AskInternally places a heavy emphasis on data privacy and security that is achieved through AI models that run completely on local machines. Moreover, it supports effecient data storage through RAG. In collaboration with the University of Waterloo, work is being done to develop multimodal knowledge graphs.",
```
Comment (sunapi386) [inline src/pages/index.astro:223]: Nice :+1:

File: src/pages/index.astro [hunk 5]
```diff
@@ -13,125 +13,118 @@ import Stats from '~/components/widgets/Stats.astro';
...
+        title: 'Strategic Market Positioning',
         description:
-          'Ensure your website looks and functions consistently across various web browsers, delivering a seamless experience to all users.',
+          'Our offline AI platform poised to dominate a significant market share.',
       },
     ]}
+    
     image={{
-      src: 'https://images.unsplash.com/photo-1519389950473-47ba0277781c?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2070&q=80',
+      src: 'https://i.ibb.co/wJPVZ0L/2a12c26e41e784706eebc097db53bab9.png',
```
Comment (sunapi386) [inline src/pages/index.astro:121]: Same as other image.

File: src/pages/index.astro [hunk 6]
```diff
@@ -13,125 +13,118 @@ import Stats from '~/components/widgets/Stats.astro';
...
+    <div class="absolute left-0 top-1/2 -mt-40 ml-[-50px] w-80 h-80 bg-blue-400 bg-opacity-40 blur-3xl rounded-full"></div>
+    <div class="absolute right-0 top-1/2 -mt-40 mr-[-50px] w-80 h-80 bg-purple-600 bg-opacity-30 blur-3xl rounded-full"></div>
+    <div class="absolute left-1/2 transform -translate-x-1/2 top-1/2 -mt-32 -ml-24 w-3/5 h-64 bg-blue-600 bg-opacity-30 blur-3xl rounded-full"></div>
+    <div class="absolute right-1/4 transform translate-x-1/3 top-1/2 -mt-32 mr-24 w-3/5 h-64 bg-blue-500 bg-opacity-30 blur-3xl rounded-full"></div>
+    <Hero class="ellipseDiv"
+      actions={[
+        {
+          variant: 'primary',
+          text: 'Get Started',
+          href: 'https://github.com/onwidget/astrowind',
```
Comment (sunapi386) [inline src/pages/index.astro:34]: Replace the actions or remove it

File: src/navigation.js [hunk 7]
```diff
@@ -2,169 +2,165 @@ import { getPermalink, getBlogPermalink, getAsset } from './utils/permalinks';
...
-        },
-      ],
-    },
-    {
-      text: 'Widgets',
-      href: '#',
-    },
   ],
-  actions: [{ text: 'Download', href: 'https://github.com/onwidget/astrowind', target: '_blank' }],
+  actions: [{ text: 'Sign In', href: 'https://github.com/onwidget/astrowind', target: '_blank' }],
```
Comment (sunapi386) [inline src/navigation.js:114]: Replace or remove the link

File: src/assets/styles/tailwind.css [hunk 8]
```diff
@@ -2,6 +2,8 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
```
Comment (sunapi386) [inline src/assets/styles/tailwind.css:5]: Preferably we don't load the fonts from google and bundle it in our resources.

**Review by sunapi386 — APPROVED** [review]
Approved but could use some changes, either in a new PR or change this one.

**Comment by jshan9078** [comment]
Looks like there a couple of typos that I am now noticing in the screenshot that I will change for the next PR.

**Comment by sunapi386** [comment]
Once a PR is ready for merge, remove the `WIP` and have a more descriptive name than `Initial Version of Frontend`. When merging, use `Squash and then merge` to keep the commit history clean

File: src/components/Favicons.astro
Comment (kevin-zkc) [inline src/components/Favicons.astro:1]: Is there any reason you import the same ~/assets/favicons/Icon.svg two times?

File: src/config.yaml
Comment (kevin-zkc) [inline src/config.yaml:1]: I thought this app is called AskInternally. Not sure if this is a typo

**Comment by sunapi386** [comment]
Merging this; @jshan9078 make a new PR if you are addressing in the future

**Pull request merged**